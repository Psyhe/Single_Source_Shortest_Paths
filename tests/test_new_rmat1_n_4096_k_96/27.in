4096 1161 1203
1161 2681 29
1161 8 89
1161 1057 112
1161 1827 85
1161 1172 43
1161 3072 219
1161 1281 235
1161 298 240
1161 320 214
1161 245 183
1161 3232 210
1161 17 40
1161 2594 6
1161 2066 87
1161 257 105
1162 712 97
1162 512 162
1162 1032 62
1162 28 210
1162 19 59
1162 1105 134
1162 530 84
1162 8 0
1162 1025 11
1162 520 104
1162 128 105
1162 2212 69
1162 65 125
1162 1301 38
1162 326 239
1162 224 194
1162 2401 121
1162 609 62
1163 332 128
1163 149 106
1163 1024 24
1163 2594 191
1163 0 209
1163 224 238
1163 3600 70
1164 1025 94
1164 17 93
1164 2226 208
1164 1 178
1164 384 50
1164 2208 247
1164 864 136
1164 32 200
1164 2754 128
1164 545 135
1164 2048 169
1164 2 54
1164 1104 71
1164 258 103
1164 1184 65
1164 17 192
1164 10 233
1164 16 131
1164 24 90
1164 1984 87
1164 0 196
1164 2563 187
1164 774 188
1164 3588 177
1164 518 9
1164 2308 83
1164 274 93
1164 2096 191
1164 608 159
1164 516 200
1164 131 156
1164 1184 124
1165 2069 47
1165 16 70
1165 3081 85
1165 24 69
1165 1152 137
1165 1041 103
1165 2562 196
1165 578 249
1166 2080 44
1166 513 76
1166 1048 28
1166 2768 85
1166 2258 122
1166 2059 35
1166 18 34
1166 1075 8
1166 32 132
1167 64 23
1167 690 128
1167 1074 232
1167 2308 35
1167 0 142
1167 770 213
1168 545 121
1168 1024 6
1168 72 43
1168 2149 60
1168 3336 215
1168 145 41
1168 276 26
1168 2116 238
1168 1 120
1168 82 62
1168 1578 139
1168 128 167
1168 1363 42
1168 128 224
1168 16 178
1168 72 174
1168 2240 203
1168 1101 227
1168 258 20
1168 2149 8
1168 2315 164
1168 2496 191
1168 520 98
1168 0 247
1168 160 27
1168 3072 91
1168 0 227
1168 2336 131
1168 3087 82
1168 2260 124
1168 1028 120
1168 275 127
1168 320 90
1168 1569 208
1168 2096 22
1168 2 249
1168 48 195
1168 6 162
1168 1 223
1168 2656 137
1168 1556 36
1168 321 165
1168 0 179
1168 2113 255
1168 4 15
1168 32 160
1168 168 239
1168 134 230
1168 73 112
1168 2588 172
1168 64 164
1168 304 253
1168 1928 61
1168 1029 243
1168 2048 251
1168 520 2
1168 2600 187
1168 824 157
1168 386 41
1168 38 139
1168 1 74
1168 2201 229
1168 16 197
1168 52 213
1168 514 76
1168 0 99
1168 456 99
1168 1120 77
1168 1032 129
1168 652 18
1168 1024 26
1168 9 169
1168 2854 113
1168 2061 103
1168 74 113
1168 2816 150
1168 384 109
1168 1041 183
1168 6 103
1168 2048 73
1169 192 172
1169 194 169
1169 271 160
1169 1601 192
1169 1153 250
1169 529 129
1169 648 114
1169 2337 55
1169 1538 21
1169 1281 192
1169 269 176
1169 1382 228
1169 2050 38
1169 1289 159
1169 2060 183
1169 434 97
1169 290 82
1169 178 248
1169 1552 119
1169 256 159
1169 2304 7
1169 1056 230
1169 66 158
1169 74 215
1170 1061 155
1170 3106 130
1170 2048 15
1170 816 243
1170 204 161
1170 3177 184
1170 454 248
1170 0 34
1170 388 230
1170 3080 146
1170 144 192
1170 9 28
1170 1025 57
1170 644 182
1170 584 34
1170 2129 108
1170 674 192
1170 1159 128
1170 64 38
1170 2304 210
1171 3528 162
1171 516 195
1171 1032 222
1171 28 224
1171 256 234
1171 3458 236
1171 65 225
1171 548 97
1171 192 112
1171 1538 140
1171 1090 202
1171 160 248
1172 46 4
1172 264 82
1172 2048 105
1172 136 19
1172 339 57
1172 1541 67
1172 8 253
1172 2058 224
1172 64 206
1172 168 78
1172 546 35
1172 1030 193
1172 5 25
1172 128 134
1172 180 170
1172 38 10
1172 4 1
1172 274 151
1172 20 126
1172 320 20
1172 448 193
1172 1161 43
1172 384 152
1172 2304 164
1172 160 7
1172 4 47
1172 593 99
1172 1057 164
1172 2049 67
1172 2645 15
1172 13 234
1172 1052 203
1172 8 45
1173 2056 149
1173 2 169
1173 264 189
1173 64 17
1173 1920 94
1174 2648 50
1174 1304 124
1174 769 177
1174 66 224
1174 1283 12
1174 400 152
1174 6 84
1174 529 241
1174 32 110
1174 2185 106
1175 528 37
1176 2224 191
1176 132 167
1176 16 86
1176 2829 38
1176 532 16
1176 512 74
1176 801 75
1176 3120 109
1176 512 9
1176 2092 66
1176 33 230
1176 140 38
1176 2626 246
1176 1104 52
1176 2081 62
1176 0 197
1176 2178 97
1176 1092 112
1176 48 137
1176 336 236
1176 1298 147
1176 2076 174
1176 587 51
1176 16 146
1177 2376 178
1177 1090 72
1177 136 133
1177 392 150
1177 906 127
1177 1025 183
1177 1025 196
1177 2304 164
1177 2244 111
1177 0 239
1178 688 133
1178 2050 98
1178 2057 194
1178 544 125
1178 36 110
1178 64 60
1178 576 62
1179 3200 180
1179 48 136
1179 1056 235
1180 2562 247
1180 2744 148
1180 152 100
1180 514 51
1180 64 92
1181 16 17
1181 1313 116
1182 1024 217
1183 2056 60
1184 360 28
1184 1538 40
1184 768 14
1184 205 228
1184 530 126
1184 546 208
1184 3088 38
1184 2104 171
1184 2120 42
1184 280 19
1184 3004 225
1184 2309 243
1184 2 232
1184 32 236
1184 3604 86
1184 2056 66
1184 596 211
1184 1537 13
1184 522 236
1184 3208 129
1184 36 10
1184 168 104
1184 1610 166
1184 129 129
1184 277 247
1184 2051 76
1184 1348 93
1184 1026 180
1184 6 199
1184 1164 65
1184 2355 97
1184 292 246
1184 1027 145
1184 1884 248
1184 208 73
1184 2560 246
1184 66 238
1184 160 223
1184 1090 84
1184 368 46
1184 32 156
1184 576 111
1184 448 60
1184 144 109
1184 626 178
1184 1032 144
1184 1038 183
1184 2181 145
1184 2584 190
1184 898 16
1184 1860 216
1184 133 204
1184 12 35
1184 2056 108
1184 156 87
1184 2016 38
1184 2058 81
1184 1024 90
1184 2306 244
1184 2562 25
1184 1024 54
1184 1 204
1184 1 208
1184 424 244
1184 2647 196
1184 576 161
1184 288 76
1184 1164 124
1184 0 148
1184 0 95
1184 136 231
1184 2050 127
1184 576 133
1184 648 9
1185 519 30
1185 2820 3
1185 3891 199
1185 772 164
1185 2054 125
1185 128 200
1185 200 109
1185 473 12
1185 2608 144
1185 1561 6
1185 2048 189
1185 34 111
1185 2368 52
1185 98 157
1185 1056 226
1185 2584 89
1185 810 48
1185 288 76
1185 104 121
1185 592 223
1186 1089 119
1186 96 243
1186 129 167
1186 8 55
1186 2057 12
1186 1288 151
1186 593 29
1186 320 252
1186 640 237
1186 576 110
1186 72 0
1186 481 134
1186 1538 176
1186 1024 110
1186 650 235
1186 1024 25
1186 1040 30
1186 554 129
1186 577 154
1186 258 145
1186 2572 103
1186 1042 77
1186 2056 121
1186 1132 43
1186 128 61
1186 1536 35
1186 516 131
1187 200 125
1187 544 45
1187 2186 73
1187 264 180
1187 128 35
1188 201 233
1188 928 82
1188 64 68
1188 641 237
1188 2049 97
1188 64 194
1188 2345 204
1188 3195 4
1188 394 50
1188 2144 221
1188 3394 109
1188 8 189
1188 8 197
1188 472 18
1188 3082 181
1188 522 118
1188 3920 89
1188 146 223
1188 96 47
1188 36 23
1188 258 71
1188 352 32
1189 0 139
1189 50 178
1189 1496 121
1189 115 161
1189 99 234
1189 2566 219
1189 1026 160
1189 2896 73
1190 1344 228
1190 21 241
1190 1040 211
1190 22 181
1190 2225 231
1190 130 243
1190 1026 139
1190 544 99
1190 32 42
1190 7 191
1191 450 13
1191 1026 133
1191 1024 226
1192 585 120
1192 2 168
1192 165 160
1192 2056 209
1192 1536 173
1192 1037 147
1192 1288 235
1192 192 13
1192 2056 93
1192 2240 178
1192 1545 0
1192 292 192
1192 780 161
1192 513 74
1192 2 45
1192 832 21
1192 970 111
1193 2050 20
1193 464 110
1193 1212 190
1193 264 98
1193 2049 109
1193 2320 159
1193 3344 249
1193 84 74
1193 2688 97
1193 548 6
1194 160 57
1194 512 143
1194 2113 147
1194 143 192
1194 1361 64
1195 24 51
1195 162 56
1196 2307 250
1196 260 189
1196 10 103
1196 550 212
1196 2052 248
1196 288 73
1196 98 218
1197 0 170
1197 2816 72
1198 136 209
1198 3712 229
1199 49 205
1199 1032 221
1200 64 44
1200 1798 84
1200 64 166
1200 0 40
1200 686 51
1200 2307 88
1200 2098 223
1200 3128 252
1200 128 238
1200 0 163
1200 256 106
1200 2118 160
1200 292 206
1200 16 50
1200 544 173
1200 2112 187
1200 392 191
1200 32 6
1200 3572 180
1200 2398 53
1200 2880 175
1200 3602 95
1200 640 239
1200 1673 21
1200 2898 121
1200 1109 254
1200 2061 193
1201 593 142
1201 296 170
1201 1090 250
1201 278 188
1201 354 248
1201 1416 33
1201 35 211
1201 1092 237
1201 2900 110
1202 8 36
1202 448 72
1202 288 226
1202 68 241
1202 212 77
1202 2338 70
1203 640 92
1203 516 239
1203 72 180
1203 3 84
1203 2704 68
