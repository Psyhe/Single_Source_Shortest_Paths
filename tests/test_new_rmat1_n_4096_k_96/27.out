13
12
9
16
42
121
88
5
11
2
70
9
19
28
49
10
66
18
30
11
65
48
107
2
18
5
9
1
47
214
24
25
60
25
121
53
1000000000000000000
38
1000000000000000000
15
65
32
1000000000000000000
